### PR TITLE
Fix player hiding being derped af

### PR DESCRIPTION
**Pull request by jakkuh**

### CODE REVIEW

**Comment by jakkuh** [comment]
Fix by developer Red_Epicness

**Comment by A5H73Y** [comment]
I'm unsure what's happening, have you abandoned the MySQL 8 SSL Error previously mentioned?

File: src/main/java/me/A5H73Y/parkour/utilities/Utils.java [hunk 1]
```diff
@@ -42,6 +21,11 @@
 import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.material.Stairs;
 
+import java.io.*;
+import java.text.Format;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
```
Comment (A5H73Y) [inline src/main/java/me/A5H73Y/parkour/utilities/Utils.java]: Please use checkstyles, this is a bad practice to import wildcards for various reasons.

File: src/main/java/me/A5H73Y/parkour/utilities/Utils.java [hunk 2]
```diff
@@ -912,9 +896,11 @@ public static void toggleVisibility(Player player, boolean override) {
     }
 
     public static void forceVisible(Player player) {
-        for (Player players : Bukkit.getOnlinePlayers()) {
-            players.showPlayer(player);
+        for (Player p : Bukkit.getOnlinePlayers()) {
+            if (Static.containsHidden(p.getName())) continue;
```
Comment (A5H73Y) [inline src/main/java/me/A5H73Y/parkour/utilities/Utils.java]: Similarly this is bad practice, please use curly brackets. Also, just invert the condition and it's the same outcome but tidier.

**Pull request closed without merging**